### ISSUE TITLE
Formula for β carbons distance

### ISSUE THREAD

**Issue by LivC182** [deepmind/deepmind-research#128]
Hi first of all congrats on you articles.

I do not fully understand how you calculate the distance between 2 residues in a PDB structure, because for the training set you need this label to be computed. 

Is it just a normal distance calculation based on the xyz coordinates of β-carbons normalized against an average distance ?

**Comment by Augustin-Zidek** [comment]
Thank you, I hope you enjoyed reading them!

The distance is the Euclidean distance between the coordinates of the atoms in the ground truth structure which is used as a target in training. We do use Cbeta atoms (Calpha for glycine, since it doesn't have a Cbeta).

As far as I am aware, we don't apply any normalisation.

**Comment by LivC182** [comment]
Ok so this means that the range of distances can be from 3-30Å, or in any case a pretty big interval.

**Comment by Augustin-Zidek** [comment]
Correct.

**Comment by Augustin-Zidek** [comment]
One more thing which might be useful to know regarding the output of the network: We place predicted distances > 22 A or < 2 A into the end bins and we divide the range 2--22 A into 64 bins.

I hope this helps. I am closing the issue for now, feel free to re-open if anything is still unclear.
